### DISCUSSION TITLE
ANY in postgres

### DISCUSSION THREAD

**Discussion by phungleson** [SeaQL/sea-query#236] [Q&A] [answered]
Hi,

I have been doing this to get rows in by ids in a vec.

```sql
"id" = ANY ('{7e4c6b15-d849-418f-ab6e-0463e873c041,...}')
```

I am trying to do something like this but it doesn't compile, as `eq` need 

```rust
pub struct Any;

impl Iden for Any {
    fn unquoted(&self, s: &mut dyn FmtWrite) {
        write!(s, "ANY").unwrap();
    }
}

.and_where(Expr::col(X::Id).eq(Func::cust(Any).arg(ids)))
```

Maybe there is something else I can do instead?

**Comment by billy1624** [comment, marked as answer]
Hey @phungleson, I think you can use `IN` operator.
- https://docs.rs/sea-query/0.20.0/sea_query/expr/struct.Expr.html#method.is_in
- https://docs.rs/sea-query/0.20.0/sea_query/expr/struct.Expr.html#method.in_subquery

**Comment by phungleson** [comment]
Yeah right, I am switching to `is_in` thanks!

**Comment by billy1624** [comment]
No problems :P
